src/Client.php

```diff
@@ -81,16 +84,16 @@ public function getOrder(string $id): object
      * Only orders with supporting capturing payment methods is allowed, for example klarna-pay-later or afterpay.
      *
```

Comment:
If capturing supports only certain payment methods, could we add a check for which payment method function `captureOrderTransaction` is called?

Reply:
We could add such check, but what we will recieve in that case?